opm/autodiff/FlowMainEbos.hpp

```diff
@@ -559,12 +624,24 @@ namespace Opm
             // create output writer after grid is distributed, otherwise the parallel output
             // won't work correctly since we need to create a mapping from the distributed to
             // the global view
-            output_writer_.reset(new OutputWriter(grid(),
+            std::vector<double> global_perms;
+            const double* writer_perms = fluidprops_->permeability();
+
+            if ( must_distribute_ )
```

Comment:
well, I don't mind much, but I think it is an additional thing which can go wrong if its not properly initialized. (and also, I always think that this initialization is terribly complicated if I see boolean attributes that are not changed after init.)